calico-3.30.yaml

```diff
@@ -686,6 +688,189 @@ subpackages:
           runs: |
             typha-client --help
 
+  - name: "calico-goldmane-${{vars.short-package-version}}"
+    description: "Goldmane"
+    dependencies:
+      provides:
+        - calico-goldmane=${{package.full-version}}
+      runtime:
+        - glibc
+        - merged-bin
+        - merged-usrsbin
+        - wolfi-baselayout
+    pipeline:
+      - uses: go/build
+        with:
+          modroot: .
+          packages: ./goldmane/cmd
+          output: goldmane
+      - uses: go/build
+        with:
+          modroot: .
+          packages: ./goldmane/cmd/flowgen
+          output: flowgen
+      - uses: go/build
+        with:
+          modroot: .
+          packages: ./goldmane/cmd/stream
+          output: stream
+      - uses: go/build
+        with:
+          modroot: .
+          packages: ./goldmane/cmd/health
+          output: health
+      - runs: |
+          mkdir -p "${{targets.subpkgdir}}"/
+          ln -sf /usr/bin/goldmane "${{targets.subpkgdir}}"/goldmane
+          ln -sf /usr/bin/health "${{targets.subpkgdir}}"/health
+          ln -sf /usr/bin/flowgen "${{targets.subpkgdir}}"/flowgen
+          ln -sf /usr/bin/stream "${{targets.subpkgdir}}"/stream
+    test:
+      environment:
+        contents:
+          packages:
+            - wait-for-it
+      pipeline:
+        - name: Test binaries
+          runs: |
+            test "$(readlink /goldmane)" = "/usr/bin/goldmane"
+            test "$(readlink /health)" = "/usr/bin/health"
+            test "$(readlink /flowgen)" = "/usr/bin/flowgen"
+            test "$(readlink /stream)" = "/usr/bin/stream"
+        - name: "start daemon on localhost"
+          uses: test/daemon-check-output
+          with:
+            start: "/goldmane"
+            timeout: 30
+            expected_output: |
+              Health enabled
+              Starting flow collector
+              Listening on
+            post: |
+              wait-for-it -t 10 --strict localhost:8080 -- echo "Server is up"
+              wait-for-it -t 10 --strict localhost:443 -- echo "Server is up"
+
+  - name: "calico-whisker-${{vars.short-package-version}}"
+    description: "Tigera OSS-UI (Whisker)"
+    dependencies:
+      replaces:
+        - nginx-mainline-config
+      provides:
+        - calico-whisker=${{package.full-version}}
+      runtime:
+        - nginx-mainline
+        - coreutils
+        - dash-binsh
+        - sed
+        - libacl1
+        - libattr1
+        - libcap
+        - libcrypt1
+        - libcrypto3
+        - libpcre2-8-0
+        - libselinux
+        - libssl3
```

Comment:
Were these dependencies not correctly detected by SCA (and generated as `so:` dependencies)?

Reply:
I suspect if it will, upstream Dockerfile particularly `apt-install` those maybe for a valid reason (i.e., runtime need etc.) Checked the final APK build here: https://apk.chaindag.dev/https/apk.cgr.dev/wolfi-presubmit/5c1e2f6cf28811954bb7df91f0036f1be5548438/aarch64/calico-whisker-3.30-3.30.0-r2.apk@etag:4cfdeebe73c5186ade0e71df0f90fa07/.PKGINFO

I able to see the `depends:` as expected. So having explicitly here would be more safer (as we already tests too!).

Reply:
> we'll run into issues in the future due to the requirements upstream changing

Yeah that makes sense. I just tested by removing those `- lib*`s and test are still passing! Means SCA already adds those. But only needed to add `libcap`, test was complaining - and they also add this lib.